src/modules/client/components/customers/CustomerProfileHeader.vue

```diff
@@ -30,9 +30,10 @@ import { NotifyPositive, NotifyNegative } from '@components/popup/notify';
 export default {
   name: 'ProfileHeader',
   computed: {
-    ...mapGetters({
-      customer: 'customer/getCustomer',
-    }),
+    ...mapGetters({ customer: 'customer/getCustomer' }),
+    deletionDisabled () {
+      return !!this.customer.firstIntervention || this.customer.contracts.length;
```

Comment:
il faut que tu mettes `!!this.customer.contracts.length` sinon `disable` recoit 0 et n'est pas content